product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/2.5-Switching-from-UDP-to-HTTP-or-HTTPS/02-synapseConfig/pom.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We do not maintain synapse config project in sub-scenario level. Shall we remove this module?